esp32/camera/motion-camera/v2-motion-camera/configmanager.h

```diff
@@ -41,14 +57,28 @@ void configSetup() {
     configManager.setAPCallback(APCallback);
     configManager.setAPFilename("index.ap.html");
 
+    configManager.setAPName(deviceName);
+
     //disable brownout detector
     WRITE_PERI_REG(RTC_CNTL_BROWN_OUT_REG, 0);
     configManager.begin(config);
+    configDefaults();
     wifiConnected = configManager.getMode() == station;
     //enable brownout detector
+
+    if (!wifiConnected) {
+      WiFi.softAPdisconnect();
+      configManager.setAPName(config.deviceName);
+      configManager.setAPPassword(config.apPassword);
+      configManager.startAP();
```

Comment:
This needed some custom [ConfigManager changes](https://github.com/nrwiersma/ConfigManager/pull/89)